test/e2e/basic/specialresource_test.go

```diff
@@ -14,8 +14,6 @@ import (
 
 const (
 	NamespaceSRO = "openshift-special-resource-operator"
-	pollInterval = 1 * time.Second
-	waitDuration = 10 * time.Minute
```

Comment:
why did you remove it from here, if the values for simple-kmod and ping-pong are the same

Reply:
@pacevedom please have a look at https://github.com/openshift-psap/special-resource-operator/pull/160 where we make all e2e tests `_test.go` files.

Reply:
In fact, should we merge #160 with this PR?